docs/content/docs/overview/glossary.md

```diff
@@ -16,53 +16,30 @@ used in the different projects.
 
 ---
 
-**Bicycle Network Analysis (BNA)**. Software that analyzes the quality and
-connectivity of bike infrastructure in cities.
+**Bicycle Network Analysis (BNA)**. Software that analyzes the quality and connectivity of bike infrastructure in cities.
 
-**City Ratings**. A PeopleForBikes program that ranks cities based on their City
-Rating generated by the BNA.
+**City Ratings**. A PeopleForBikes program that ranks cities annually based on their ratings generated by the BNA.
 
-**BNA Analysis**. A set of actions that is used to produce the BNA Output for a
-single city.
+**Analysis**. A set of actions that is used to produce a rating for a single city.
 
-**BNA Output**. The set of files and data generated by the BNA Analysis for a
-single city.
+**Submission**: A quadruplet consisting of a city name, state, country, and FIPS code, provided to initiate an analysis in the BNA. A Submission can have three approval statuses: Pending, Approved, or Rejected.
 
-**City Rating**. A score out of 100 points representing the quality and
-connectivity of the bike network in a given area (typically a city) generated by
-the BNA. The City Rating is one value contained within a city's BNA Output.
+**BNA Component**. Ouput values generated for a city by running the BNA. Some but not all components are used to calculate a city's overall score. 
```

Comment:
I would say that the definition is more related to the "BNA Output".

A component is a part of a rating, like a subsection of the results generated by the BNA.

In the code, we divided a rating into the following 8 components:
- Summary
- Core Services
- Infrastructure
- Opportunity
- People
- Recreation
- Retail
- Transit

Reply:
I added your other definition of BNA Output and updated BNA Component with a definition similar to what you provided above.